src/software/new_geom/util/physics.cpp

```diff
@@ -0,0 +1,23 @@
+#include "software/new_geom/util/physics.h"
+
+Point calculateFuturePosition(const Point &initial_position,
+                              const Vector &initial_velocity, const Vector &acceleration,
+                              double time_in_future)
```

Comment:
The thought here is that all of these params are unitless with the assumption that the units make sense. See the comment in physics.h

Reply:
Ah I see. I still think `Duration` is preferable here because it's a bit more clear it's a time-based value, but it can be somewhat "unitless" as well since it can be constructed with seconds or ms, and get either value as well.

For these functions as well as all the private helpers, I think using Duration instead of `double` will just be a bit more clear and easy to use. Given that our code is pretty much exclusively MKS units I don't see a huge benefit to making this really generic.

@EvanMorcom @garethellis0 @akhilveeraghanta @kelvinkoon ?

Reply:
I think using `Duration` makes sense here.

Reply:
ok changed

Reply:
I don't see the change?

Reply:
whoops changed now